### PR TITLE
NONE Just a way share my comments from the documentation review

### PR DESCRIPTION
**Pull request by agorina**
I am going to abandon this PR, it is just my way to share the feedback.

### CODE REVIEW

File: docs/Architecture.md [hunk 1]
```diff
@@ -22,16 +23,54 @@ The RDW uses other systems from the SBAC Open Test System environment, including
...
The deployment takes advantage of the AWS ecosystem including the following technologies:
-
* **Aurora**. High performance Amazon version of MySQL. Backs both the data warehouse and the reporting data mart.
-* **Redshift**. Amazon OLAP database.
* **ElastiCache**. Redis-compatible memory cache used by the web apps to cache user session information.
* **S3**. Reliable, long-term storage used to archive all data that enters the system.
* **ELB**, **Route 53**, **Certificate Manager** for load balancing, routing and SSL termination.

Here is an overview of the RDW processes:
+To make deployment easier, all the RDW applications have been containerized. The orchestration of the applications is handled with Kubernetes. The current deployment is in **AWS** so kops is used to configure the many resources needed by the cluster.
```
Comment (agorina) [inline docs/Architecture.md]: I am not sure why it thinks this changed.

File: docs/Architecture.md [hunk 2]
```diff
@@ -22,16 +23,54 @@ The RDW uses other systems from the SBAC Open Test System environment, including
...
+
+The deployment takes advantage of the AWS ecosystem including the following technologies:
+
+* **Aurora**. High performance Amazon version of MySQL. Backs both the data warehouse and the reporting data mart.
+* **Redshift**. Amazon OLAP database.
+* **ElastiCache**. Redis-compatible memory cache used by the web apps to cache user session information.
+* **S3**. Reliable, long-term storage used to archive all data that enters the system.
+* **ELB**, **Route 53**, **Certificate Manager** for load balancing, routing and SSL termination.
+
+Here is an overview of the RDW processes:
```
Comment (agorina) [inline docs/Architecture.md]: Ditto.

File: docs/Architecture.md [hunk 3]
```diff
@@ -22,16 +23,54 @@ The RDW uses other systems from the SBAC Open Test System environment, including
...

**[Report Processor](Runbook.md#report-processor)**. This processor generates PDF reports. It is horizontally scalable and many instances should be run to deal with reporting load.  

**[PDF Generator](Runbook.md#pdf-generator)**. This application converts HTML to PDF. It is used by the report processor. It is horizontally scalable and many instances should be run to deal with reporting load. 

**[Admin Web App](Runbook.md#admin-webapp)**. This web application is used by customers to manage student groups. It is horizontally scalable with each process handling ??? concurrent users. Because student group management is infrequent, # instances should be sufficient.
+**RabbitMQ**. The message queue is an infrastructure service (like the database or config repository). However it is deployed in a container as part of the cluster so it is called out here. Note that the queue is not used as the sole storage for any critical data so an HA configuration is good for system reliability but not necessary for data integrity.
```
Comment (agorina) [inline docs/Architecture.md]: Ignore this section too, there is no change.

Comment (agorina) [inline docs/Architecture.md]: Ignore this section too, there is no change.

File: docs/Architecture.md [hunk 4]
```diff
@@ -22,16 +23,54 @@ The RDW uses other systems from the SBAC Open Test System environment, including
...
+* **Redshift**. Amazon OLAP database.
+* **ElastiCache**. Redis-compatible memory cache used by the web apps to cache user session information.
+* **S3**. Reliable, long-term storage used to archive all data that enters the system.
+* **ELB**, **Route 53**, **Certificate Manager** for load balancing, routing and SSL termination.
+
+Here is an overview of the RDW processes:
 
 ![RDW Processes](rdw-processes.png)
 
+Nit: it would help to have an annotation explaining that 'horizontal cylinders' represent a message queue or wrap it into a box with MQ as a header (it is done this way in some other places)
```
Comment (malaffoon) [inline docs/Architecture.md]: I've tried adding/enhancing a legend and i've yet to come up with something that is effective and not distracting.

File: docs/Architecture.md [hunk 5]
```diff
@@ -9,6 +9,7 @@ The RDW is divided into sub-systems based on data, performance and scalability r
 * **Data Warehouse**. This is where data enters the system. It is the collection of all assessment definitions, institutions, students, student groups, and test results. It includes supplemental reference data such as subject claims and traits, common core standards, accessibility/accommodation codes and descriptions, ethnicities, and grades.
 * **Reporting**. This is the main reporting data mart and web UI. The test results have been optimized for presentation to teachers and administrators to allow them to quickly view results.
 * **Aggregate Reporting**. This is the system for viewing test results through the lens of aggregation, trends, etc. The test results have been optimized for slicing and dicing.
+* **Admin**. This is the system to perform various administrative tasks, such as ...
```
Comment (malaffoon) [inline docs/Architecture.md]: added

File: docs/Architecture.md (hunk 3 above)
Comment (malaffoon) [reply docs/Architecture.md]: Ignored.

Well, i guess not really  ;)

File: docs/Runbook.migrate.md [hunk 6]
```diff
@@ -41,7 +41,8 @@ There are core tables that are created as part of the initial schema and are not
 - migrate
 
 The `migrate` table is the main control table for the migrate. It stores the migrate status and the timestamp range of content handled by each migrate job. The migrate process is managed by the “migrate-reporting” service.
-  
+The service migrates data in chunks using **created** and **updated** timestamps in the `master` tables. The starting and ending timestamps for each chunk are derived from the **import** table.
```
Comment (malaffoon) [inline docs/Runbook.migrate.md]: added

File: docs/Runbook.md [hunk 7]
```diff
@@ -69,7 +69,7 @@ The import service is the REST end-point for submitting data to the system. It i
 #### Configuration
 The [Annotated Configuration](../config/rdw-ingest-import-service.yml) describes the properties and their effects.
 
-
+Just a comment: I am not sure if there is a better way to handle it, but I am struggling with the duplicate information because we have to keep it in synch.
```
Comment (malaffoon) [inline docs/Runbook.md]: Yup, that's an issue. Trying to make each markdown document at least partly stand on its own while avoiding duplicate information is tough.

File: docs/Monitoring.md [hunk 8]
```diff
@@ -13,6 +13,8 @@ Monitoring RDW applications includes monitoring:
 ### Database 
 There are a number of tables that provide useful information about the state of the system.
 
+Should we add a reference to a newly created validation script?
```
Comment (malaffoon) [inline docs/Monitoring.md]: added

File: docs/API-Ingest.md [hunk 9]
```diff
@@ -266,6 +266,8 @@ curl -X POST --header "Authorization:Bearer {access_token}" -F file=@winterICA.1
   https://import-service/exams/imports
 ```
   
+Should we say something about update/delete exams?
```
Comment (malaffoon) [inline docs/API-Ingest.md]: added

**Comment by malaffoon** [comment]
@agorina thanks for reviewing the docs. I'll post a PR with my changes.

File: docs/Architecture.md (hunk 4 above)
Comment (agorina) [reply docs/Architecture.md]: I like how it is done in import-service.png.

**Pull request closed without merging**